backend/app/main/views.py

```diff
@@ -198,3 +198,62 @@ def send_mail():
 
     send_email(recipients, subject)
     return "message sent!"
+
+# get all donors from Airtable
+@main.route("/donors", methods=["GET"])
+def get_all_donors():
+    response = requests.get(
+        "https://api.airtable.com/v0/appw4RRMDig1g2PFI/Donors",
+        headers={"Authorization": str(os.environ.get("API_KEY"))},
+    )
+    response_json = response.json()
+    list_of_donors = []
+    for r in response_json["records"]:
+        name = r["fields"].get("Name")
+        notes = r["fields"].get("Notes")
+        email = r["fields"].get("Donor Email")
+        donations = r["fields"].get("Total Donated")
+        if name is not None:
+            m = Donor(name=name, email=email, notes=notes, total_donated=donations)
+            list_of_donors.append(m.serialize())
+    return jsonify(list_of_donors)
```

Comment:
Consider adding error checking to see if the response is valid, on error return an error code instead of an empty list. This change might not be strictly necessary

Reply:
Same as above